src/main/java/com/opentok/SipProperties.java

```diff
@@ -0,0 +1,174 @@
+/**
+ * OpenTok Java SDK
+ * Copyright (C) 2018 TokBox, Inc.
+ * http://www.tokbox.com
+ *
+ * Licensed under The MIT License (MIT). See LICENSE file for more information.
+ */
+package com.opentok;
+
+/**
+ * Defines values for the <code>properties</code> parameter of the
+ * {@link OpenTok#sipDial(String, String, SipProperties)}  method.
+ *
+ * @see OpenTok#sipDial(String, String, SipProperties)
+ */
+public class SipProperties {
+    private String sipUri = null;
+    private String from = null;
+    private String userName = null;
```

Comment:
I think this should be `username`

Reply:
Just following the camelCase convention here.